resources/views/layout/main.blade.php

```diff
@@ -138,13 +156,12 @@
     </div>
     <footer class="bg-dark text-center text-light pb-4">
         <div class="d-flex flex-column justify-content-center align-items-center">
-            <p class="mt-5"><a class="text-decoration-underline text-light" href="https://projects.hanisirfan.xyz/seajell" target="_blank">SeaJell</a> {{ $appVersion }}</p>
-            <p>Hak Cipta &copy; <a href="http://hanisirfan.xyz" class="text-light">Muhammad Hanis Irfan bin Mohd Zaid</a> 2021</p>
+            <p class="mt-5"><a class="text-decoration-underline text-light"
+                    href="https://projects.hanisirfan.xyz/seajell" target="_blank">SeaJell</a> {{ $appVersion }}</p>
+            <p>Hak Cipta &copy; <a href="http://hanisirfan.xyz" class="text-light">Muhammad Hanis Irfan bin Mohd
+                    Zaid</a> 2021</p>
         </div>
     </footer>
-    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-U1DAWAznBHeqEIlVSCgzq+c9gqGAJn5c/t99JyeKa9xxaYpSvHU5awsuZVVFIhvj" crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.14.0-beta2/dist/js/bootstrap-select.min.js"></script>
-    @bukScripts(true)
```

Comment:
I think you forgot to the the JS back from `layout.footer` . All the JS dependent stuff isn't working.

Reply:
Ouh snap. forgot about it

Reply:
Hahaha no problem. Btw, can you change back to the old login page design? I've asked for a feedback from someone else about the design (my sister actually), she said that my design is better.

Reply:
@hanisirfan , I think you have the right not to approve the merge right? My code doesn't affect the main branch until you approve.

Reply:
Yes I do. But you have actually committed a lot of stuff I felt wrong not approving it. It's just this design that I think should be changed.

Reply:
Oh maybe, you can use the design you made but just add the logo at the top.